module/VuFind/tests/unit-tests/src/VuFindTest/Security/CspHeaderGeneratorTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/**
+ * CspHeaderGenerator test
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+namespace VuFindTest\Security;
+
+use VuFind\Security\CspHeaderGenerator;
+
+/**
+ * CspHeaderGenerator test
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class CspHeaderGeneratorTest extends \PHPUnit\Framework\TestCase
+{
+    use \VuFindTest\Feature\FixtureTrait;
+
+    /**
+     * Nonce generator mock
+     *
+     * @type \PHPUnit\Framework\MockObject
```

Comment:
For properties @var tag should be used. Also, could you add the real class to typehint? Something like this: `\PHPUnit\Framework\MockObject&\VuFind\Security\NonceGenerator`? It says a bit better what the type of property is I think.

Reply:
Thanks @xmorave2 -- this is fixed.